src/bot/create_bot.py

```diff
@@ -48,11 +48,17 @@ def create_bot(post_storage: PostStorage, warden: Warden) -> TelegramClient:
         bot_token=TELEGRAM_BOT_TOKEN
     )
 
-    # TODO: should we parse album here?
+    # TODO: should we parse album here? nea
```

Comment:
Why nea?

Reply:
For what purpose? To take a name if it's not necessary?